packages/clerk-js/src/utils/url.ts

```diff
@@ -362,3 +362,16 @@ export const mergeFragmentIntoUrl = (_url: string | URL): URL => {
 export const pathFromFullPath = (fullPath: string) => {
   return fullPath.replace(/CLERK-ROUTER\/(.*?)\//, '');
 };
+
+const frontendApiRedirectPaths: string[] = [
+  '/oauth/authorize', // OAuth2 identify provider flow
+  '/v1/verify', // magic links
+  '/v1/tickets/accept', // ticket flow
+];
```

Comment:
@mzhong9723 should we also add `/choose` or other redirect urls? What urls should be contained here?

Reply:
These are only FAPI paths for FAPI initiated redirection flows